src/cloudinary.js

```diff
@@ -686,7 +687,7 @@ class Cloudinary {
               setUrl = false;
             }
           }
-          const isLazyLoading = (options.loading === 'lazy' && !this.isNativeLazyLoadSupported() && this.isLazyLoadSupported() && !elements[0].getAttribute('src'));
+          const isLazyLoading = (options.loading === 'lazy' && isIntersectionObserverSupported() && !tag.getAttribute('src'));
```

Comment:
@maoznir can you explain the change here?

Reply:
@strausr 
1. Remove check of native lazy loading, we will not rely on it and use intersection observer always.
2. isIntersectionObserverSupported() is a different name for isLazyLoadSupported(), which was a duplicate implementation.
3. tag instead elements[0], because it seems wrong to check the first element for all elements